lib/standard/string.nit

```diff
@@ -1957,6 +2025,11 @@ redef class Int
 	# Returns a string describing error number
 	fun strerror: String do return strerror_ext.to_s
 
+	# Converts an `Int` to its `UnicodeChar` equivalent
+	#
+	# `self` is supposed to be a code point in unicode.
+	fun to_unicode_char: UnicodeChar is extern "intcp_to_u8char"
```

Comment:
I know that I am weird, but for me `to_unicode_char` should behave in a way conherent with `to_f` or `to_s` that is translating the value not just mapping it.

~~~
print 9 # 9
print 9.to_f # 9.0
print 9.to_s # 9
print 9.to_unicode_char # got tabulation (codepoint 0x9), expected '9' (codepoint 0x39)
~~~

I woult rename this method `to_code_point` (or `as_code_point`) to be clear that this just maps an interger to the char with the associated code point value